Automattic/sensei
# Keep current status for a question when it is updated

Resolves #7601 

When we update a question, its status resets to "draft".

The issue appears only for questions that are not in use (are not part of a quiz or of a category).

## Proposed Changes
* Preserve the post status for a question when it is being updated.

## Testing Instructions
1. Go to Sensei LMS -> Questions.
2. Create a new question and publish it.
3. Open the question in the editor. (Probably, it is already open in the editor for you.)
4. Modify the question content and click "Update".
5. Make sure you didn't see the notification with text "Post is reverted to draft" and the post (question) is still published.

## Pre-Merge Checklist
